src/imp/noop_imp.js

```diff
@@ -1,18 +1,25 @@
+function NoopSpanContextImp() {
```

Comment:
nit: why all of these called `Imp`, including the package?  What's wrong with `NoopSpan`, `NoopTracer` ?